_posts/2024-05-28-release - Copy.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Release 2024.22.1
+author: Priya
+date: 2024-05-31
+category: Jekyll
+layout: post
+---
+### Enhancement:
+
+* **Default Naming Improvements:** Updated the default project name to “New project” and the default workspace name to “firstname’s Workspace” for a more polished user experience.
+* **Enhanced Sign-Up Process:** Introduced an organization field to the sign-up form, streamlining account setup for professional users.
+* **Cleaner Workspace Names:** Eliminated word breaks in workspace names, ensuring a seamless and professional display.
```

Comment:
This would go in the defects section
Introduced word breaks in the workspace names on sidebar.